tests/cross/src/test/scala/tests/pc/InlineValueSuite.scala

```diff
@@ -213,6 +213,141 @@ class InlineValueSuite extends BaseCodeActionSuite with CommonMtagsEnrichments {
     InlineErrors.notLocal,
   )
 
+  checkEdit(
+    "for-comprehension",
+    """|object Main {
+       |val a =
+       |  for {
+       |    i <- List(1,2,3
```

Comment:
```suggestion
       |    i <- List(1,2,3)
```
looks like there is a compile error here :thinking: